storybook/stories/core/components/Input/RadioPillGroup.stories.js

```diff
@@ -0,0 +1,295 @@
+import { actions } from '@storybook/addon-actions'
+
+import RadioPillGroup from '@orangehrm/oxd/core/components/Input/RadioPills/RadioPillGroup'
+import { h, ref } from 'vue'
+
+const templateDecorator = () => ({
+  template: `
+<div><story/></div>
+	`,
+})
+
+export default {
+  title: 'Inputs/RadioPillGroup',
+  component: RadioPillGroup,
+  argTypes: {
+    name: {
+      control: { type: 'text' },
+      table: {
+        type: { summary: 'name for the radio pill group' },
```

Comment:
Better to have the first letter capital since it's shown in the documentation

Reply:
Done